wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -82,3 +95,45 @@ def test_cleaned_fields(self):
         self.assertIsInstance(last_published_at, datetime)
         self.assertIsInstance(latest_revision_created_at, datetime)
         self.assertEqual(wp_link, "")
+
+
+@override_settings(
+    BASE_URL="http://localhost:8000",
+    WAGTAIL_WORDPRESS_IMPORT_CATEGORY_PLUGIN_ENABLED=True,
+    WAGTAIL_WORDPRESS_IMPORT_CATEGORY_PLUGIN_MODEL="example.models.Category",
+)  # testing requires a live domain for requests to use, this is something I need to change before package release
+# mocking of somesort, using localhost:8000 for now
+class WordpressItemImportTests(TestCase):
+    from example.models import Category
+
+    fixtures = [
+        f"{FIXTURES_PATH}/dump.json",
+    ]
+
+    def setUp(self):
+        self.importer = WordpressImporter(f"{FIXTURES_PATH}/raw_xml.xml")
+        self.logger = Logger(LOG_DIR)
+        self.importer.run(
+            logger=self.logger,
+            app_for_pages=IMPORTER_RUN_PARAMS_TEST["app_for_pages"],
+            model_for_pages=IMPORTER_RUN_PARAMS_TEST["model_for_pages"],
+            parent_id=IMPORTER_RUN_PARAMS_TEST["parent_id"],
+            page_types=IMPORTER_RUN_PARAMS_TEST["page_types"],
+            page_statuses=IMPORTER_RUN_PARAMS_TEST["page_statuses"],
+        )
+
+        self.parent_page = Page.objects.get(id=IMPORTER_RUN_PARAMS_TEST["parent_id"])
+        self.imported_pages = self.parent_page.get_children().all()
+
+    def test_category_snippets_are_saved(self):
+        snippets = Category.objects.all()
+        self.assertEqual(len(snippets), 4)
+
+    def test_page_has_categories(self):
+        page_one = self.imported_pages.get(title="Item one title")
+        page_one_categories = page_one.specific.categories.all()
+        self.assertEqual(2, page_one_categories.count())
+
+        page_two = self.imported_pages.get(title="Item two title")
+        page_two_categories = page_two.specific.categories.all()
+        self.assertEqual(2, page_two_categories.count())
```

Comment:
**Suggestion**: Add a test that short category names are ignored if below the min length setting.